posts/2022-09-29-22.0.0.11-beta.adoc

```diff
@@ -0,0 +1,174 @@
+
+You can activate the features incorperating the following in your `server.xml`:
+
+[source, xml]
+----
+   <featureManager>
+       <feature>servlet-6.0</feature>
+   </featureManager>
+----
+
+The Jakarta Servlet 6.0 Specification and Javadocs can be found link:https://jakarta.ee/specifications/servlet/6.0/[here].
+
+
+// https://github.com/OpenLiberty/open-liberty/issues/22115
+[#connectors]
+== Connectors-2.1
+
+This feature allows Open Liberty users that want to use the latest release of the link:https://github.com/eclipse-ee4j/jca-api[Jakarta resource specification] within their application.
+
+The main update to connectors to come out of this feature is the addition of Java generics support to the jakarta.resource.cci.ResourceFactory, jakarta.resource.cci.IndexedRecord, and jakarta.resource.cci.MappedRecord classes 
+
+Open Liberty is keeping our connectors implementation up to date with the latest specification.
+
+You can activate the features incorperating the following in your `server.xml`:
+[source, xml]
+----
+<server>
+ <featureManager>
+   <feature>connectors-2.1</feature>
+ </featureManager>
+</server>
+----
+
+All other `resourceAdapter` configurations within server.xml can remain unchanged for current applications.
+
+You can find out more about this specific update below:
+
+* link:https://github.com/eclipse-ee4j/jca-api[Spec Project]
+* link:https://mvnrepository.com/artifact/jakarta.resource/jakarta.resource-api[Spec API]
+* link:https://jakarta.ee/specifications/connectors/2.1/apidocs/jakarta.resource/module-summary.html[Spec Javadoc]
+
+
+// https://github.com/OpenLiberty/open-liberty/issues/22078
+[#persistence]
+== Persistence 3.1
+
+Jakarta Persistence API is a richly featured API which provides object-model approach to persisting, fetching, and modifying data stored on a Relational Database System.
+
+JPA has been around since EE 5, but before JPA you had to use EJB's Bean or Container managed entities, or raw JDBC to interact with database storage.
+
+
+[#run]
+=== Try it now 
+
+To try out these features, just update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 19, Java SE 18, Java SE 17, Java SE 11, and Java SE 8.
+
+// // // // // // // //
+// In the preceding section:
+// Check if a new non-LTS Java SE version is supported that needs to be added to the list (17, 11, and 8 are LTS and will remain for a while)
+// // // // // // // //
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+  <groupId>io.openliberty.beta</groupId>
+  <artifactId>openliberty-runtime</artifactId>
+  <version>22.0.0.11-beta</version>
+  <type>pom</type>
+</dependency>
+----
```

Comment:
does the regular beta include full InstanOn support? I only ask because we have a dedicated container image for it and why would you need that if the regular beta supports InstantOn?

Reply:
That's a good question which I expect the InstantOn standalone blog post to cover.  In short, kind of.... the liberty runtime (what those maven coordinates are for) does include the checkpoint feature.  However, for InstantOn to be functional, certain libraries have to also be available on the OS, which is why we have the InstantOn specific container images (aka, they provide both the Liberty beta runtime, and the necessary CRIU libraries that the OS needs to use the beta functionality).